bot.js

```diff
@@ -1,37 +1,34 @@
-require('dotenv').config();
+const { config } = require('dotenv');
+const { fetchNextEventForGroup, formatEventMessage } = require("./meetupHelpers");
 const { Client, Intents } = require('discord.js');
-const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
 
-const fetchMeetupData = async () => {
-	// fetch stuff here
-};
+config();
+
+const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
 
 const sendAnnouncement = async (channel) => {
-	await fetchMeetupData();
-	channel.send('data fetched');
+	const event = await fetchNextEventForGroup(34547654);
```

Comment:
Handle the case where event is null.

Reply:
@fredlawl Do you think we should throw an error here or just console log that there aren't any upcoming events and return undefined?